src/app/auth/forgot_password/[token]/page.tsx

```diff
@@ -47,27 +48,14 @@ export default function ResetPassword() {
 
   return (
     <div className="flex h-screen items-center justify-end bg-[url(/images/calendar.svg)] bg-center bg-repeat">
-      <div className="absolute top-4 left-6 flex items-center gap-1 select-none sm:gap-2">
-        <svg
-          xmlns="http://www.w3.org/2000/svg"
-          width="24"
-          height="24"
-          viewBox="0 0 24 24"
-          fill="none"
-          stroke="currentColor"
-          strokeWidth="2"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-          className="lucide lucide-bird-icon lucide-bird stroke-primary-400 size-14"
-        >
-          <path d="M16 7h.01" />
-          <path d="M3.4 18H12a8 8 0 0 0 8-8V7a4 4 0 0 0-7.28-2.3L2 20" />
-          <path d="m20 7 2 .5-2 .5" />
-          <path d="M10 18v3" />
-          <path d="M14 17.75V21" />
-          <path d="M7 18a6 6 0 0 0 3.84-10.61" />
-        </svg>
-        <span className="text-3xl font-bold">pombo</span>
+      <div className="absolute top-0 left-6 flex items-center gap-1 select-none sm:gap-2">
+        <Image
+          src="/images/logo.svg"
+          alt="Pombo Logo"
+          width={64}
+          height={64}
+          className="h-20 w-40"
```

Comment:
I think you can define just one of these two and it will scale accordingly, right?